config/sim_blue.cfg

```diff
@@ -0,0 +1,17 @@
+[GAME]
+type=sim
+our_color=blue
+autonomous_play=false
+coach_fps=10
+
+[COMMUNICATION]
+type=sim
+field_port_file=config/field/sim.cfg
+ui_debug_address = 127.0.0.1
+grsim_udp_address = 224.5.23.2
+grsim_port=20011
```

Comment:
instead of `grsim_port`, could it be `grsim_cmd_port` or `grsim_cmd_sender_port`, so it's not confuse with the vision port?

Reply:
oups merge too fats :(